docs/tutorials/fundamentals/viewer.md

```diff
@@ -382,109 +360,126 @@ from scipy import ndimage as ndi
+   * **Median** - the median projection takes the median pixel intensity for the final projected image.   
+* **Split RGB** - if the image layer is an RGB image, it will be split into 3 new layers with red, green, and blue values in separate layers. 
+* **Split Stack** - if an image layer is a stack (has 3 or more dimensions), it is split into a list of layers along the axis. This option takes a little time to execute. Properties will be changed as follows: 
+    * **Colormap:** (magenta, green) for a stack with 2 channels, (CYMRGB) for stacks with more than 2 channels  
+    * **Blending:** additive  
+    * **Contrast_limits:** min and max values of the layer
+    * All other properties, such as **Scale** and **Translate** will be propagated from the original stack.   
+* **Merge to Stack** - combines a set of layers to a single-layer stack. The resulting layer stack will contain the layers with their original ordering in the layer list. Layers must be of the same type (e.g. An **Image** layer can be merged only with other **Image** layers.) and must have the same dimensionality.  (e.g. a 1024 x 1024 layer can only be merged with another 1024 x 1024 layer.)
+* **Link Layers** - links the selected layers.  Once layers are linked, any action performed on one layer will be performed on all linked layers at the same time. The layer control panel will show _only_ when a single layer is selected. Changing properties with that layer's control panel will change properties in all of the linked layers.  
+* **Unlink Layers** - appears when layers are linked. It unlinks the layers so that changes to one of the layer's properties no longer result in the same changes to the previously linked layers. 
+* **Select Linked Layers** - appears only when layers are linked. Selects all layers linked to a given layer.  
 
 +++
 
-## Changing viewer theme
+## Changing the viewer theme
 
-Currently, **napari** comes with two different themes `light` and `dark`, which
-is the default. To change the theme, update `theme` property of the viewer:
+Currently, **napari** comes with `light`, `dark` themes for the viewer;  the default is `dark`. Additionally, there is the `system` virtual theme that will attempt to match the viewer theme (`light` or `dark`) to your system theme on macOS, Windows, and some Linux. To change the preferred theme used for all viewers you can use the **Preferences** menu item in the **File** or **napari** menu and then select the **Appearance** tab. You can also change the `theme` property of the *current* viewer by using the following code:
 
-```{code-cell} python
+```{code-cell} python  
 from skimage import data
 
 import napari
 
 viewer, image_layer = napari.imshow(data.astronaut(), name='astronaut')
 
-# change the viewer theme
+# change the viewer theme  
 viewer.theme = 'light'
 ```
 
 ```{code-cell} python
-:tags: [hide-input]
+:tags: [hide-input]  
+
 nbscreenshot(viewer, alt_text="A napari viewer changed to light theme")
 ```
```

Comment:
This screenshot isn't showing for me in the built docs, but again I notice the one above doesn't have the tag to hide output, so maybe it's that?
(it does show with live docs on napari.org)

Reply:
This one never did show.  The next time you see the viewer, it has a black background again. If you think we should, I can take out the part that says "hide-input" so you can see the command to change it back.  It will add only a couple of lines to the doc.